src/common/store/pdfLink/actions.js

```diff
@@ -0,0 +1,19 @@
+import { createAction } from 'redux-actions';
+import { uploadLink } from '../../services/fileService';
+
+export const setLink = createAction('PDF_LINK_ADD');
+
+export const setError = createAction('PDF_LINK_ERROR');
+
+export const setName = createAction('PDF_LINK_NAME');
```

Comment:
You can reuse action for file `updatePdfFile`. There is no need to store name and id seperatly for links